src/Adapter/Requirement/CheckRequirements.php

```diff
@@ -112,7 +120,11 @@ private function getErrorMessages()
             'log_dir' => $this->translator->trans('Set write permissions for the "log" folder and subfolders.', array(), 'Admin.Advparameters.Notification'),
             'mails_dir' => $this->translator->trans('Set write permissions for the "mails" folder and subfolders.', array(), 'Admin.Advparameters.Notification'),
             'module_dir' => $this->translator->trans('Set write permissions for the "modules" folder and subfolders.', array(), 'Admin.Advparameters.Notification'),
-            'theme_lang_dir' => $this->translator->trans('Set the write permissions for the "themes%s/lang/" folder and subfolders, recursively.', array('%s' => _THEME_NAME_), 'Admin.Advparameters.Notification'),
+            'theme_cache_dir' => $this->translator->trans('Set the write permissions for the "themes/%s/cache/" folder and subfolders, recursively.', array('%s' => _THEME_NAME_), 'Admin.Advparameters.Notification'),
```

Comment:
Sorry guys for the delayed review, long weekend... Can't we write _Set write permissions_ instead of _Set the write permissions_ to have all the wordings harmonized? Otherwise domains are good here. ;-)

Reply:
Nice catch, updated.